localization/english/ancient_magic_decisions_l_english.yml

```diff
@@ -5,7 +5,52 @@
  forge_material_soul_decision_desc: "Bodily immortality is one thing, but the brain can only handle so much. Stuff hundreds of years of knowledge into one mind, and you simply run out of room. To reach true immortality, one needs to reforge their soul into a limitless receptable of knowledge, so that one can truly transcend one's mortal coil!"
  forge_material_soul_decision_confirm: "Let's forge my soul"
  MAGIC_DEMENTIA_IMMUNITY: "If successful, provides immunity to [GetTrait('mage_dementia').GetName( GetNullCharacter )]"
+ 
+ implant_magical_blood_decision: "Implant Blood"
+ implant_magical_blood_decision_tooltip: "I have some blood in my inventory, that could be used to boost my own body."
+ implant_magical_blood_decision_desc: "With the extracted blood i got, i could go boyond my base innate mana affinity potential."
+ implant_magical_blood_decision_confirm: "Lets begin"
+ AM_implant_magical_blood_choose_type: "Choose type of Blood"
+ AM_implant_magical_blood_option_close_family: "Blood of [close_family|E]"
+ AM_implant_magical_blood_option_close_family_tooltip: "Magical blood extracted from [close_family_members|E]. #S High Compatibility #!"
```

Comment:
Not to a native English speaker. Language isn't about typing the words that make sense to you and hoping everyone else understands, it's about typing the words that maximize everyone else's chance to understand. You wouldn't put French in the English localization, because it'd be confusing to English speakers. Putting random capitalization is less confusing, but still confusing.

Each time I see "Blood" instead of "blood," I have to remind myself that they are the same thing, because so many stories use "Blood" to refer to a specific group of people, and not to blood at all. That should also not be capitalized, unless it's a game mechanic.

On a similar note, I've been having trouble trying to determine if Mana should be capitalized every time it's shown. There's no distinct non-capitalized mana for it to get confused with, though, so I mostly let it be.

Reply:
yes i get that Blood is name and blood is blood, Its the same in my language, but there is something else that i also take into consideration, and that is that sometimes things aren't about whats right gramaticaly but what looks better, its the same with transaltion, sometimes you cant exactly translate something, because in that language it makes no sense or it looks or sounds terrible in that moment.

Reply:
and about mana, i think it doesnt matter as mana is still the same with with M or m in my opinion, but im not english speaker so...

Reply:
You're free to ignore the note, but just remember: the notes are here, given from a native English linguist to a non-native speaker and from a professional programmer to a student, in order to help. Ignoring them only makes the mod worse.